static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx

```diff
@@ -110,8 +92,11 @@ class IssueWidgetQueries extends Component<Props, State> {
 
   componentWillUnmount() {
     this.unlisteners.forEach(unlistener => unlistener?.());
+    this._isMounted = false;
   }
 
+  private _isMounted: boolean = false;
```

Comment:
Added this because I noticed some noise in the test output regarding calling setState after a component unmounted. We already do this in `widgetQueries.tsx` and `releaseWidgetQueries.tsx`